ceph_deploy/mon.py

```diff
@@ -232,11 +232,9 @@ def mon_add(args):
 
 
 def mon_create(args):
-
     cfg = conf.ceph.load(args)
     if not args.mon:
-        mon_initial_members = cfg.safe_get('global', 'mon_initial_members')
-        args.mon = re.split(r'[,\s]+', mon_initial_members)
+        args.mon = get_mon_initial_members(args, _cfg=cfg)
 
     if not args.mon:
         raise exc.NeedHostError()
```

Comment:
If we pass error_on_empty=True to get_mon_intial_members() just above this, this raise can go, correct?  Looks that way to me.